client/src/pages/addFlight/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './addflight.css';
+import Durationcard from '../../components/durationcard';
+import Select from '../../components/select';
+import Input from '../../components/input';
+import Timebutton from '../../components/timebutton';
+import Button from '../../components/button';
+import Header from '../../components/header';
+import Sidenav from '../../components/sidenav';
```

Comment:
What I want you to do here is simple, require all these modules in `/component/` in an `index.js`, then export them, and in this file destructure that file so that your imports are done in a single import.